pkg/odo/cli/component/common_push.go

```diff
@@ -17,19 +17,18 @@ import (
 
 // CommonPushOptions has data needed for all pushes
 type CommonPushOptions struct {
-	show             bool
+	// TODO(feloy) Fixme
+	showFlag         bool //nolint:structcheck
 	componentContext string
-	pushConfig       bool
-	pushSource       bool
+	configFlag       bool
+	sourceFlag       bool
```

Comment:
I didn't want to document at this place because it is already documented during the declaration of the flag:
```
	pushCmd.Flags().BoolVar(&po.configFlag, "config", false, "Use config flag to only apply config on to cluster")
	pushCmd.Flags().BoolVar(&po.sourceFlag, "source", false, "Use source flag to only push latest source on to cluster")
```

In this specific case, it is in another file (push.go), but generally, it is declared in the same file.